examples/example.yaml

```diff
@@ -40,9 +40,9 @@ spec:
       memory: 8Gi
   tlsEnabled: true
 # You can set either a version of the db or a specific image name
-# cockroachDBVersion: v21.1.7
+# cockroachDBVersion: v21.1.9
   image:
-    name: cockroachdb/cockroach:v21.1.7
+    name: cockroachdb/cockroach:v21.1.9
```

Comment:
Did you modify the template that generates this file? Most of our YAML files are generated to update with the latest version.